dkms.in

```diff
@@ -866,11 +866,12 @@ prepare_signing()
             ubuntu* )
                 sign_file="/usr/src/linux-headers-$kernelver/scripts/sign-file"
                 ;;
-            fedora* | rhel* | ovm*)
-                sign_file="/lib/modules/$kernelver/build/scripts/sign-file"
-                ;;
             * )
-                echo "Binary sign-file not found, module won't be signed"
+                if [ -f "/lib/modules/$kernelver/build/scripts/sign-file" ]; then
+                    sign_file="/lib/modules/$kernelver/build/scripts/sign-file"
+                else
+                    echo "Binary sign-file not found, module won't be signed"
```

Comment:
I think this should be unconditional here, and this message should be in the new check below.

Reply:
Right, here is my opinion about having a case for different distributions shinning trough. I think it would be better to search the sign-file in the 3 possible places, and avoid trying to know the specifics of the distributions and derivatives.

Anyway, for this change it makes sense to only set the variable.